ansible/tasks/check_strapi_image_exists-test-prod.yaml

```diff
@@ -0,0 +1,18 @@
+---
+  - name: Fetch imagestream {{ react_infra_name }}:{{ version }}
+    set_fact: 
+      imagestreamtag: "{{ lookup('k8s', kind='ImageStreamTag', namespace=image_namespace, resource_name=strapi_infra_name + ':' + version) }}"
+    until: (imagestreamtag|length > 0)
+    delay: 60
+    retries: 20
+    ignore_errors: True
+  - name: Image Tag {{ version }} does not exist in {{ image_namespace }}. 
+    debug: 
+      msg: "Image Tag {{ version }} does not exist in {{ image_namespace }}. \
+           This is because either you have not run ansible-playbook build-react.yaml -e PR={{ PR }} \
```

Comment:
This message should be a referencing a different playbook

Reply:
Yes, forgot to update the debug message. Will update.